### DISCUSSION TITLE
Remove Edit brush button

### DISCUSSION THREAD

**Discussion by pravdomil** [keiwando/toonsquid#249] [Ideas]
clicking on the brush and then clicking on edit is sufficient enough

the button is too wide, and B is in lower case
and if I select different category then it is not clear what brush I am editing
also it will be more clean
<img width="416" alt="image" src="https://user-images.githubusercontent.com/2387356/230770045-043626e8-fbad-4c7f-84df-179c114e26ae.png">

**Comment by Cat-Butler** [comment]
I disagree with your second point. The lower-case b is correct.
In several places, ToonSquid differentiates between commands (Hide layer, Edit brush) and labels/names (Animation Clip, Blend Mode).
If that was a brush that did some sort of editing (an Edit Brush, or a Blur Brush), then upper-case B would be correct.

The button is an excellent efficiency shortcut when creating / modifying a brush (I've been playing with creating some custom brushes), reducing the number of taps when editing a brush, trying it on the canvas, editing the brush some more, trying it again, etc. The pop-up menu adds an extra tap at each step.
As such, I'd prefer it to remain unless/until the brush panel becomes too cluttered from extra features.

**Reply by pravdomil** [reply]
files app for reference
<img src="https://user-images.githubusercontent.com/2387356/233589751-dd705c2c-a7a0-4131-9b09-811f6c53e958.jpeg" width="320">

so leave the button but make it smaller not the full width of the container

**Reply by Cat-Butler** [reply]
Yes, I would be happy with that.

**Reply by pravdomil** [reply]
also all the commands elsewhere should be written with Title Case (see the Files app for reference)

**Reply by Cat-Butler** [reply]
On this issue I personally still prefer the ToonSquid approach, if applied consistently - it connects better with the way I think. That does not mean my preference is correct.
